### ISSUE TITLE
support for command for creating env with `conda env create --name <env_name> <python_version>` without requirement of environment.yml file 

### ISSUE THREAD

**Issue by VikasPoddar** [conda/conda#12479]
### Checklist

- [X] I added a descriptive title
- [X] I searched open requests and couldn't find a duplicate

### What is the idea?

# Conda Environment creation command
<p>
In conda we usually create environment via these two command often : 
</p>

```
conda create --name <env_name> python=<version>
```

```
conda env create --file environment.yml
```
first command do not feel as **intuitive** as it should be and second command feel intuitive but second command only works if _yml file_ is present.
It is not a big problem or in some sense not problem at all for seasoned users.

but for maintaining logical consistency in command following command must work as above two commands works
```
conda env create --name <env_name> python=<version>

```
I think will really going to be helpful for every user as it will make the commands more logical.

>for example : 
_github-cli_ uses same logical order
* gh repo <sub_command>

I think conda must have this pattern
```
conda env <sub_command>
```
for creating new environment from scratch.

Please forgive me if I have said anything wrong, this is my first _issue_ so' I don't much about how to put good issue.
Thank you

### Why is this needed?

Conda user either regular user or newbie, everyone going get benefit from this feature.

 * increase speed since logical order and no need to remember the exception 
 * user may find not remember the command, as the command present itself in logically in mind

### What should happen?

It may increases the logical consistency in conda command, helping user to logical drive the command.
for example : 

``` 
conda create --name <env_name> <python-version>
```
It is little confusing for a new users to understand what is begin created.

while following command
``` 
conda env create --name <env_name> <python-version>
```
It will be going to be more explicitly and intuitive for developers to understand command in one read.
It will increase UX 10x .

**Comment by travishathaway** [comment]
Hi @VikasPoddar,

Thanks for submitting this feature request. We actually have an ongoing discussion about the user experience related to conda env commands here:

- https://github.com/conda/conda/issues/11633

We welcome you to join that discussion if this is something you feel strongly about. I have also linked this issue there as well.

I just want to let you know though that we are currently considering completely removing the `conda env` commands entirely and just collapsing everything into conda. You do bring up some good counter points to this though, which is why I linked this issue there.
